modules/core/src/main/java/org/apache/ignite/internal/management/cache/VerifyBackupPartitionsTaskV2.java

```diff
@@ -140,27 +157,6 @@ public class VerifyBackupPartitionsTaskV2 extends ComputeTaskAdapter<CacheIdleVe
         }
     }
 
-    /**
-     * @param results Received results of broadcast remote requests.
-     * @return Idle verify job result constructed from results of remote executions.
-     */
-    public static IdleVerifyResultV2 reduce0(List<ComputeJobResult> results) {
```

Comment:
Why do you remove this method?

Reply:
Oh, was public static for an external usage. Forgot